management/server/core/hub-manager/hub-manager-impl/src/main/java/io/subutai/core/hubmanager/impl/HubEnvironmentManager.java

```diff
@@ -365,9 +406,23 @@ public Integer call() throws Exception
         }
     }
 
+    public EnvironmentManager getEnvironmentManager()
+    {
+        return environmentManager;
+    }
+
 
     protected CompletionService<PrepareTemplatesResponseCollector> getCompletionService( Executor executor )
     {
         return new ExecutorCompletionService<>( executor );
     }
+
+
+    public RequestBuilder getSetupP2PConnectionCommand( String interfaceName, String localIp, String p2pHash,
+                                                        String secretKey, long secretKeyTtlSec )
```

Comment:
why do you need this command here? this is internal thing exposed by LocalPeer interface appropriate methods